frontend/leaderboard/src/Types.re

```diff
@@ -25,11 +25,27 @@ module Metrics = {
     | TransactionsSent
     | SnarkWorkCreated;
 
+  type metricRecord = {
+    blocksCreated: option(int),
+    transactionSent: option(int),
+    snarkWorkCreated: option(int),
+  };
+
   let stringOfMetric = metric => {
     switch (metric) {
-    | BlocksCreated => "blocks_created"
-    | TransactionsSent => "transactions_sent"
-    | SnarkWorkCreated => "snarkwork_created"
+    | Some(BlocksCreated) => "blocks_created"
+    | Some(TransactionsSent) => "transactions_sent"
+    | Some(SnarkWorkCreated) => "snarkwork_created"
+    | None => ""
+    };
+  };
```

Comment:
This one being from option(metric) to string is kinda funny, could be left as is and then do 
```
optionalMetric
|> Option.map(stringOfMetric)
|> Option.value(~default="")
```
if we needed to calculate it on an optional one